metricbeat/module/nginx/stubstatus/stubstatus.go

```diff
@@ -0,0 +1,72 @@
+// Reads server status from nginx host under /server-status, stub_status module is required.
+package stubstatus
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+
+	"github.com/elastic/beats/metricbeat/helper"
+	_ "github.com/elastic/beats/metricbeat/module/nginx"
```

Comment:
I'm thinking about moving this line also to have it as part of `list.go`. But that is not an issue of this PR but one on our side.